docs/integration/alerts-telegram-integration.md

```diff
@@ -0,0 +1,85 @@
+description: Sematext threshold, anomaly and / or heartbeat Alerts integration with Telegram messenger. Use them together to get the information about each notification quickly and on any device.
+
+## In Telegram
+
+**1.** Create a Telegram account at [telegram.org](https://telegram.org).
+
+**2.** Setup your Telegram client and start chatting with a user named **BotFather**. This is a place where you can create your bot. Just type in:
+
+```
+/newbot
+```
+
+And in the next line type your bot name, i.e. **SematextNotificationBot**. 
+
+Finally, type the bot username, it has to end with the **_bot** suffix, i.e. **SematextNotificationBot_bot**.
+
+If everything went successfully you should get a message with the **Bot Token API key**:
+
+<img class="content-modal-image" alt="Create Telegram Bot Token API Key" src="../../images/integrations/create-telegram-integration_bot_key.png" title="Create Telegram Bot Token API Key">
+
+Note the **Bot Token API key** down, you will need to provide it in Sematext UI.
+
+**3.** Invite the bot to the channel it should send the notifications to. You can do that by including the bot in the administrators of the channel in your Telegram client. Let's follow the steps. 
+
+First, message the bot using the Telegram client of your choice:
+
+<img class="content-modal-image" alt="Create Telegram Integration - Message Bot" src="../../images/integrations/create-telegram-integration_bot_message.png" title="Create Telegram Integration - Message Bot">
+
+Next, click the channel name:
+
+<img class="content-modal-image" alt="Create Telegram Integration - Click Channel Name" src="../../images/integrations/create-telegram-integration_click_channel_name.png" title="Create Telegram Integration - Click Channel Name">
+
+Click the **Administrators** to list the channel administrators:
+
+<img class="content-modal-image" alt="Create Telegram Integration - List Administrators" src="../../images/integrations/create-telegram-integration_add_administrator.png" title="Create Telegram Integration - List Administrators">
+
+Click the **Add Admin** button:
+
+<img class="content-modal-image" alt="Create Telegram Integration - Add Admin" src="../../images/integrations/create-telegram-integration_add_new_administrator.png" title="Create Telegram Integration - Add Admin">
+
+Find the created bot and click on it:
+
+<img class="content-modal-image" alt="Create Telegram Integration - Add the Bot" src="../../images/integrations/create-telegram-integration_add_bot.png" title="Create Telegram Integration - Add the Bot">
+
+Finally review the permissions and click done:
+
+<img class="content-modal-image" alt="Create Telegram Integration - Review Permissions" src="../../images/integrations/create-telegram-integration_add_bot_finish.png" title="Create Telegram Integration - Review Permissions">
+
+**4.** Retrieve the channel identifier that the bot will send the notifications to. This can be done by running the following request:
```

Comment:
Retrieve the channel identifier that the bot will send the notifications to
==>
Retrieve the channel identifier for the channel where the bot will send notifications.

Reply:
Done.